packages/dashboard/src/components/sidePanel/sections/aggregationSection/helpers.tsx

```diff
@@ -0,0 +1,40 @@
+import type { SelectProps } from '@cloudscape-design/components';
+import { AggregateType } from '@aws-sdk/client-iotsitewise';
+
+export const aggregateToString = (aggregate?: string): string => {
+  return aggregate ? aggregate.replace(/_/g, ' ').toLowerCase() : 'auto';
```

Comment:
It's not immediately obvious why there is the call to `replace`. Do you think a comment would help?